Code/ThirdParty/LivePP/init.cmake

```diff
@@ -2,21 +2,25 @@
 # NOTES: To not use github storage, you can download the Live++ SDK from the official website and put it in the same directory as this file.
 
 if(EZ_CMAKE_PLATFORM_WINDOWS)
-    set(EZ_SUPPORT_LIVEPP ON CACHE BOOL "Enable Live++ support for Ez")
+    set(EZ_3RDPARTY_LIVEPP_SUPPORT OFF CACHE BOOL "Enable Live++ support for Ez")
 else()
-    set(EZ_SUPPORT_LIVEPP OFF CACHE BOOL "Enable Live++ support for Ez")
+    set(EZ_3RDPARTY_LIVEPP_SUPPORT OFF CACHE BOOL "Enable Live++ support for Ez")
 endif()
 
 # #####################################
 # ## ez_requires_livepp()
 # #####################################
 macro(ez_requires_livepp)
     ez_requires_one_of(EZ_CMAKE_PLATFORM_WINDOWS)
```

Comment:
ez_requires_one_of(EZ_CMAKE_PLATFORM_WINDOWS)

This should be redundant, because EZ_3RDPARTY_LIVEPP_SUPPORT is already tied to the platform.

Reply:
Fixed in latest commit.